tests/Predis/Command/Redis/OBJECT_Test.php

```diff
@@ -92,6 +92,19 @@ public function testObjectEncoding(): void
         $this->assertMatchesRegularExpression('/[zip|quick]list/', $redis->object('ENCODING', 'list:metavars'));
```

Comment:
As per my test, only `quicklist` can happen here.

Also `[zip|quick]` should be `(zip|quick)` and have `^` and `$` if you mean to check for ziplist or quicklist